.idea/django_phone_verify.iml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why are these files needed?

Reply:
Sorry forgot to ignore these files. I will do that in the follow-up commit.